Docs/Jython-Modules.md

```diff
@@ -197,6 +197,43 @@ core.link.remove_link("Kodi_Control")
 ```
 </ul>
 
+#### Module: [`core.metadata`](..Core/automation/lib/python/core/metadata.py)
+<ul>
+
+This module allows simpler interaction with item metadata. There are two instance classes that can be used manage a single namespace, or multiple namespaces of a single item.
```

Comment:
In an md, put sentences on separate lines with a space after the period.

Reply:
I'll make the changes, but purely out of curiosity, why?

Reply:
If there's one big blob of text on a line, it's harder to track changes. openhab-docs requires per sentence tracking too. I think there might even be a SAT check for this.